python/mxnet/contrib/onnx/_import/import_model.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+"""import function"""
+# pylint: disable=no-member
+
+from .import_onnx import GraphProto
+
+def import_model(model_file):
+    """Imports the supplied ONNX model file into MXNet symbol and parameters.
+    :parameters model_file
+    ----------
+    model_file : ONNX model file name
+
+    :returns (sym, params)
```

Comment:
Check for doc rendering. Will be helpful if we decide to add these API docs on the website later.
Follow the same convention throughout